test/dist_autograd_test.py

```diff
@@ -99,9 +100,15 @@ def _exec_func(self, exec_mode, method, *args):
             if len(args) == 1 and isinstance(args[0], list):
                 return method(*args[0])
             return method(*args)
-        else:
+        elif ExecMode.RPC_SYNC == exec_mode:
             return rpc.rpc_sync('worker{}'.format(self._next_rank()), method,
                                 args=(args))
+        elif ExecMode.REMOTE == exec_mode:
+            rref = rpc.remote('worker{}'.format(self._next_rank()), method,
+                              args=(args))
+            return rref.to_here().wait()
+        else:
+            raise ValueError("Unrecognized ExecMode {}".format(exec_mode))
```

Comment:
This is awesome! Thanks for refactoring dist_autograd_test in such a clean way to enable RRef tests.